index.css

```diff
@@ -70,7 +70,7 @@
   background-color: var(--button-col);
   border: 5px solid var(--button-col);
   border-radius: 3rem;
-  box-shadow: 0.3rem 0.3rem rgb(0,0,0,0.2);
+  box-shadow: 0.2rem 0.2rem rgb(0,0,0,0.2);
   outline: none;
```

Comment:
`outline: none` should be avoided as it creates an accessibility issue. In place of this, we can write custom CSS for `:focus`, `:hover`, etc